processing/flare_capa/flare_capa.py

```diff
@@ -24,30 +25,70 @@ class FlareCapa(ProcessingModule):
 
     def initialize(self):
         if not HAVE_CAPA:
-            raise ModuleInitializationError(self, 'Missing dependency: flare-capa')
+            raise ModuleInitializationError('Missing dependency: flare-capa')
```

Comment:
```suggestion
            raise ModuleInitializationError(self, 'Missing dependency: flare-capa')
```

As discussed in https://github.com/certsocietegenerale/fame_modules/issues/68 , `ModuleInitializationError()` take two arguments